### ISSUE TITLE
training codes of loss functions

### ISSUE THREAD

**Issue by sureJ0824** [h4nwei/SPAQ#5]
Great Works.
I am really interested in the subwork of multi-task learning from scene labels，and I think it may of great help for my work. However, it may be really difficult for me to reproduce its training codes, especially about the loss functions, as in  your test codes I can't find it, could you please release the training code of multi-learning from scene labels.
Thank you  sincerely

**Comment by h4nwei** [comment]
Dear sureJ0824 @sureJ0824 ,

Sorry. We are not going to release the training code. For the loss function, we trained the network to predict the log variance, log(\sigma_1) and log(\sigma_2). If there are some other problems with the reproduction, please let me know.

Best, 
Hanwei
